cypress/integration/UserCanViewProducts.feacture.js

```diff
@@ -0,0 +1,10 @@
+beforeEach(() => {
+  cy.intercept("GET", "**/api/products", {
+    fixture: "products.json",
+  }).as("getProducts");
+  cy.visit("/");
+});
+
+it("is expected to display a list with 3 items", () => {
+  cy.get("[data-cy=ProductList]").children().should("have.length", 3);
```

Comment:
please use `snake_case` or `kebab-case` in data attributes. In thes case `product_list` or `product-list`

Reply:
> please use `snake_case` or `kebab-case` in data attributes. In thes case `product_list` or `product-list`

changed to kebab-case